src/core_plugins/console/public/src/es.js

```diff
@@ -21,6 +21,7 @@ module.exports.send = function (method, path, data, server, disable_auth_alert)
   var options = {
     url: '../api/console/proxy?uri=' + encodeURIComponent(path),
     data: method == "GET" ? null : data,
+    contentType: 'application/json',
```

Comment:
We need to assign this based on the actual content type, as of now there are two content types possible, `application/json` for standard bodies and `text/plain` for newline-delimited json bodies for APIs like [bulk](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-bulk.html)